app/controllers/Tournament.scala

```diff
@@ -240,7 +233,7 @@ object Tournament extends LilaController {
                 isGranted(_.ManageTournament)) 1 else 4
               CreateLimitPerUser(me.id, cost = 1) {
                 CreateLimitPerIP(HTTPRequest lastRemoteAddress ctx.req, cost = 1) {
-                  env.api.createTournament(setup, me, teams) flatMap { tour =>
+                  env.api.createTournament(setup, me, teams, getUserTeamIds) flatMap { tour =>
```

Comment:
since we already have `teams` here we could just pass `_ => fuccess(teams)` to save a DB call, but it's only on tournament creation so we don't care.

Reply:
Kinda... they are a different datatype though. Could perhaps do something like that but with teams.zip etc.?

Reply:
`getUserTeamIds` is `User => Fu[List[TeamId]]` right? So should be `_ => fuccess(teams.map(_.id))`

Reply:
Well, teams isn't a Map but a List of a (TeamId, TeamName) so ._1 it will be. Not sure why I just thought about zip. Will do all this probably tomorrow.

Reply:
Yeah, don't bother. Tournament creation is rare enough